patches/postcss-url.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/.editorconfig b/.editorconfig
```

Comment:
Shesh, what a pain this must have been!

Reply:
It was. At first I thought I'd have to fork the whole repo, but pnpm provides a nice way to patch a dependency without having to fork the whole deal. I think this should be fine as this npm package hasn't been updated in 4 years. It works, just is unmaintained (but there are no real other alternatives).